sklearn/tree/tests/test_export.py

```diff
@@ -210,6 +210,26 @@ def test_graphviz_toy():
                 'fillcolor="#e5813900"] ;\n' \
                 '}'
 
+    # Test Gradient Boosting Classifier
+    clf = GradientBoostingClassifier(n_estimators=2, random_state=0)
+    clf.fit(X, y)
+
+    contents1 = export_graphviz(clf.estimators_[0][0], filled=True,
+                                out_file=None)
+    contents2 = 'digraph Tree {\n' \
+                'node [shape=box, style="filled", color="black"] ;\n' \
+                '0 [label="X[1] <= 0.0\\nfriedman_mse = 0.25' \
+                '\\nsamples = 6\\nvalue = 0.0", fillcolor="#e5813980"] ;\n' \
+                '1 [label="friedman_mse = 0.0\\nsamples = 3' \
+                '\\nvalue = -2.0", ' \
+                'fillcolor="#e5813900"] ;\n' \
+                '0 -> 1 [labeldistance=2.5, labelangle=45, ' \
+                'headlabel="True"] ;\n' \
+                '2 [label="friedman_mse = 0.0\\nsamples = 3\\nvalue = 2.0", ' \
+                'fillcolor="#e58139ff"] ;\n' \
+                '0 -> 2 [labeldistance=2.5, labelangle=-45, ' \
+                'headlabel="False"] ;\n' \
+                '}'
     assert_equal(contents1, contents2)
```

Comment:
Btw, this is the important line, and you've stolen it from the previous test, so we're no longer testing that case...

Reply:
Ok, I removed it. And added a test for the case that input argument is not estimator.